api/graphql/schema.graphql

```diff
@@ -1069,6 +1079,12 @@ type Query {
   myAuth: UserAuthInfo @auth
   me: User @auth
   user(id: UUID! @eq): User @find @softDeletes @guard @canQuery(ability: "view")
+  govEmployee(
```

Comment:
Usually the query name matches (or is close to) the type it returns.  Could we get closer with this?

Reply:
Right, good catch. Updated in [rename type and query](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12847/commits/0c60b716655f506fa9f05ed90e2614f00be3f783)